contributing/Dockerfile

```diff
@@ -1,27 +1,25 @@
-FROM ubuntu:18.04
+ARG NODE_VERSION=16.15.1
+FROM node:${NODE_VERSION}-slim
 
-ARG NODE_VERSION=8.10.0
-ARG GO_VERSION=1.18.1
+ARG GO_VERSION=1.18.3
 ARG BUD_VERSION=main
 
-# Install basic dependencies
-RUN apt-get -qq update
-RUN apt-get -qq -y install curl git make gcc g++
-
-# Install Node.js
-RUN curl https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.gz | tar -xz -C /usr/local --strip-components 1
 RUN node -v
 
+# Install basic dependencies
+RUN apt-get -qq update \
+  && apt-get -qq -y install wget git make gcc g++ \
+  && rm -rf /var/lib/apt/lists/*
+
 # Install Go
-RUN curl -L https://golang.org/dl/go$GO_VERSION.linux-amd64.tar.gz | tar -xz -C /usr/local --strip-components 1
+RUN wget -q -O - https://go.dev/dl/go${GO_VERSION}.linux-amd64.tar.gz | tar -xz -C /usr/local --strip-components 1
```

Comment:
Any reason for wget over curl?

Reply:
Based on your link, it seems like we could update this to `--output -`? 

I have a slight preference towards curl for no other reason than I'm a lot more familiar with it.

Reply:
Hmm... thought I gave it a shot, but must have mixed up something back then. `curl` seems to be working fine with this solution, so let's just keep it.